tests/ui/test_ui_tools.py

```diff
@@ -1405,6 +1405,31 @@ def test_invalid_color_format(self, mocker, color):
                          count=5)
         assert str(e.value) == "Unknown color format: '{}'".format(color)
 
+    @pytest.mark.parametrize('stream_id, muted_streams, called_value', [
+        (86, {}, 204),
+        (86, {86, 205}, -1),
+        (205, {14, 99}, 0),
+        pytest.param(99, {14, 86}, 0, marks=pytest.mark.xfail)
```

Comment:
@neiljp What do you think of these types of parametrize, which test for failing conditions?